recipes/r-picante/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set version = '1.6-2' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-picante
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: picante_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/picante_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/picante/picante_{{ version }}.tar.gz
+
+  sha256: 4db3a5a0fe5e4e9197c96245195843294fbb8d0a324edcde70c6ab01276ab7ff
+
+build:
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-ape
+    - r-nlme
+    - r-vegan
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+
+  run:
+    - r-base
+    - r-ape
+    - r-nlme
+    - r-vegan
+    - libgcc  # [not win]
+
+test:
+  commands:
+    - $R -e "library('picante')"  # [not win]
+    - "\"%R%\" -e \"library('picante')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=picante
+  license: GPL-2
+  summary: Phylocom integration, community analyses, null-models, traits and evolution in R
+  license_family: GPL2
```

Comment:
This does not seem to be the latest version of the helper script. Can you please add license_file as well?

Reply:
sure